service/resource/vaultpki/create_test.go

```diff
@@ -0,0 +1,129 @@
+package vaultpki
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/giantswarm/certificatetpr"
+	"github.com/giantswarm/micrologger/microloggertest"
+	"github.com/giantswarm/vaultpki/vaultpkitest"
+	vaultapi "github.com/hashicorp/vault/api"
+)
+
+func Test_Resource_VaultPKI_GetCreateState(t *testing.T) {
+	testCases := []struct {
+		Obj           interface{}
+		CurrentState  interface{}
+		DesiredState  interface{}
+		ExpectedState VaultPKIState
+	}{
+		{
```

Comment:
Some comments on these test cases would be appreciated

Reply:
Done.